### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by t-bltg**
```bash
$ (cd doc; bash gen_imgs.sh)
```
, now generates most of the doc examples for `README.md`, to keep it up-to-date with the latest changes.

I've modified some examples to use `border = :dotted` as a workaround for https://github.com/JuliaPlots/UnicodePlots.jl/issues/9.

### CODE REVIEW

File: doc/gen_imgs.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+{
+  ver=2.x
+
+  ${JULIA-julia} gen_imgs.jl $ver
+
+  for f in imgs/$ver/*.txt; do
+    html=${f%.txt}.html
+    cat $f | ${ANSI2HTML-ansi2html} --input-encoding=utf-8 --output-encoding=utf-8 >$html
+    ${WKHTMLTOIMAGE-wkhtmltoimage} --quiet --crop-w 800 --quality 85 $html ${html%.html}.png
```
Comment (johnnychen94) [inline doc/gen_imgs.sh:10]: Good to know this 👍

**Review by johnnychen94 — APPROVED** [review]
Nice!

At some point, we might reach large repo storage and make package downloading slower. As you may have noticed: some image files are permalinks to githubusercontent.com: people use this trick to not add the contents into git repos.

This PR will double the repo size. Another trick to work around this is to keep the shell scripts in `gh-pages` branch, keep the image data there, and use permalinks to target the images there. The Julia Pkg system only downloading contents from the commit and not the entire repo (say, `master` branch) so putting things to `gh-pages` won't make it slower for users to do `pkg> add UnicodePlots`

```console
$ du -sh
184K	doc
4.0K	LICENSE.md
4.0K	Manifest.toml
4.0K	Project.toml
 16K	README.md
212K	src
1.4M	test
```

This storage stuff isn't urgent at all so whether doing this trick totally depends on how you like it.

**Comment by t-bltg** [comment]
> At some point, we might reach large repo storage and make package downloading slower. As you may have noticed: some image files are permalinks to githubusercontent.com: people use this trick to not add the contents into git history so as to get a thin repo.
> 
> This PR will double the repo size. Another trick to work around this is to keep the shell scripts in `gh-pages` branch, keep the image data there, and use permalinks to target the images there. The Julia Pkg system only downloading contents from the commit and not the entire repo (say, `master` branch) so putting things to `gh-pages` won't make it slower for users to do `pkg> add UnicodePlots`

I totally omitted the repo size when `DL`ing `UnicodePlots`, I only checked for the size of individual images displaying in the `README.md`.
Those are very relevant remarks !

This PR is only a workaround in some sense, I think this is cumbersome to use `Documenter` for the docs, and also, there are plot rendering issues as in `http://docs.juliaplots.org/latest/generated/unicodeplots/` which looks sparse between lines.

My preference to reduce the repo size will be to move the docs to another branch.
Is it mandatory to use `gh-pages`, I'm afraid we can accidentally remove those contents ?
Maybe we could use `unicodeplots-docs` ?

**Comment by johnnychen94** [comment]
unicodeplots-docs sounds good to me. This isn't an urgent issue so really depends on if you want to future-proof this right now or fixes it in the future when someone complains it.

File: doc/gen_imgs.sh (hunk 1 above)
Comment (t-bltg) [reply doc/gen_imgs.sh:10]: This is hacky, but was the only sequence that allowed me to generate correct pngs.

`ansi2html` does a good job, but then the issue is to include `html` in a `markdown`, and I haven't found a way to do that, so I've defaulted to `png` conversion.

**Comment by t-bltg** [comment]
> unicodeplots-docs sounds good to me. This isn't an urgent issue so really depends on if you want to future-proof this right now or fixes it in the future when someone complains it.

I'll do it right now, doubling the repo size is not acceptable for doc only (even if the initial repo size is small :wink:)

**Comment by t-bltg** [comment]
Ok, it's done. I've added a note on how to update doc imgs: https://github.com/t-bltg/UnicodePlots.jl/tree/doc_imgs#documentation-update

**Pull request merged**